docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# InputFieldWrapper
+
+This component is used to wrap input fields such as
+[InputText](../?path=/docs/components-forms-and-inputs-inputtext-docs--page) and
+[InputPressable](../docs/components-forms-and-inputs-inputpressable-docs--page)
```

Comment:
Last thing - I think this InputPressable and the one on line 35 are missing the path part, they're broken

Reply:
Oh god, I copied the wrong part, thanks for catching it